source/sessions/driver-sessions.md

```diff
@@ -710,7 +710,8 @@ expiration.
 
 ## Gossipping the cluster time
 
-Drivers MUST gossip the cluster time when connected to a deployment that uses cluster times.
+Drivers MUST gossip the cluster time when connected to a deployment that uses cluster times. However, drivers MUST NOT
+gossip cluster time on SDAM commands.
```

Comment:
Should this link to the SDAM spec in some way, so that it's clear what SDAM commands refers to?

Reply:
I followed the pattern in the load balancer spec and added "SDAM" to the terms section.